tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmRestApiIT.java

```diff
@@ -57,20 +57,22 @@ static void baseTearDown() {
     @AfterEach
     @Timeout(value = 5, unit = TimeUnit.MINUTES)
     void tearDown() throws Exception {
-        HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME,
-                "--cascade",
-                "foreground",
-                "--namespace",
-                NAMESPACE);
-        K8sUtil.waitForAllPodsDeletedInNamespace(client, NAMESPACE);
-        HELM_CHART_CONTAINER.deleteNamespace(NAMESPACE);
-
-        HELM_CHART_CONTAINER.uninstallRelease(OPERATOR_RELEASE_NAME,
-                "--cascade",
-                "foreground",
-                "--namespace",
-                "default");
-        K8sUtil.waitForAllPodsDeletedInNamespace(client, "default");
+        try {
+            HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME,
+                    "--cascade",
+                    "foreground",
+                    "--namespace",
+                    NAMESPACE);
+            K8sUtil.waitForAllPodsDeletedInNamespace(client, NAMESPACE);
+            HELM_CHART_CONTAINER.uninstallRelease(OPERATOR_RELEASE_NAME,
+                    "--cascade",
+                    "foreground",
+                    "--namespace",
+                    "default");
+            K8sUtil.waitForAllPodsDeletedInNamespace(client, "default");
```

Comment:
If it's better to wait for all pods to be gone, should we maybe integrate that into the `deleteNamespace()` method? Although in this test case we won't delete the `default` namespace, so we'd probably still need this as extra method.

Not sure what the best balance between separation of concern and convenience is here.

Reply:
Actually you're right, this method `waitForAllPodsDeletedInNamespace` is only needed for the [HelmNonRootUserInstallPlatformIT](https://github.com/hivemq/helm-charts/blob/feature/20810-private-keystore-password/tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmNonRootUserInstallPlatformIT.java) class since we are not deleting the namespace there, but I think it's better to refactor the latter test class to keep the same consistency across ITs regarding the initial setup (beforeEach, afterEach, afterAll, BeforeAll), so I will just delete the namespace and we can get rid of this util method `waitForAllPodsDeletedInNamespace`.